hackyourtemperature/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.send('hello from backend to frontend!');
+});
+
+app.post('/weather', (req, res) => {
+  const {cityName} = req.body;
+  if(!cityName){
+    res.send('City Name is required!');
+  } else {
```

Comment:
@ErhanKRL Great work! we don't need else is not necessary here. 


 ```suggestion
  if(!cityName){
    res.send('City Name is required!');
  }
  res.send(`You submitted: ${cityName}`);
```

Reply:
@tarektarho I tried to delete the else statement but it gives 
"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client" error when I tested with postman. I think it should stay(it doesn't give error with else) because it continues to execute the next line and tries to send it again.

Reply:
Yes, let's revers it  then :) 


```suggestion
app.post("/weather", (req, res) => {
  const { cityName } = req.body;
  if (cityName) {
    res.send(`You submitted: ${cityName}`);
  }
  res.send("City Name is required!");
});
```

Reply:
Aha. Got it. Thanks.